test/binance_test.exs

```diff
@@ -38,9 +38,24 @@ defmodule BinanceTest do
       assert info.server_time != nil
 
       assert info.rate_limits == [
-               %{"interval" => "MINUTE", "limit" => 1200, "rateLimitType" => "REQUESTS"},
-               %{"interval" => "SECOND", "limit" => 10, "rateLimitType" => "ORDERS"},
-               %{"interval" => "DAY", "limit" => 100_000, "rateLimitType" => "ORDERS"}
+               %{
+                 "interval" => "MINUTE",
+                 "limit" => 1200,
+                 "rateLimitType" => "REQUEST_WEIGHT",
```

Comment:
I had a feeling that this is a "breaking" change as somebody could potentially rely on `"rateLimitType" => "REQUESTS"` - it does not have impact on this PR but it will have an impact when we will cut new release (this is comment/note to myself :wink: )